granary/meetup.py

```diff
@@ -65,7 +65,14 @@ def _create(self, obj, preview=False, include_link=source.OMIT_LINK, ignore_form
         else:
             return self.return_error('Meetup.com syndication does not support %(verb)s' % {'verb': verb})
 
-        event_url = obj.get('inReplyTo')
+        # parse the in-reply-to out
+        url_containers = obj.get('object')
```

Comment:
mind switching this to `self.base_object(obj)`? i think you could then also drop lines 70-81.

https://github.com/snarfed/granary/blob/b2b1bf7c6a66d9192a48bfc7d8a5e74e246ab0e9/granary/source.py#L845-L880

Reply:
Done - I seem to still need the lines, otherwise invalid URLs don't get rejected properly :+1: